src/autopr/LibInfo.d

```diff
@@ -270,22 +272,29 @@ struct LibInfo
     ***************************************************************************/
 
     bool markSupported ( bool complete, int maintained_minor_versions,
-        int maintained_major_months, Release[] releases )
+        int maintained_major_months, Release[] releases, bool d2_only )
     {
         import std.range;
         import std.algorithm;
         import std.datetime;
 
-        auto onlyRCorFull ( Release rel )
+        auto d2filter ( Release rel )
+        {
+            return d2_only == rel.ver.metadata.canFind("d2");
```

Comment:
Will also match `v1.2.3+r2d2` :)

This highlights deficiency in our `Version` module btw - there can be several metadata blocks, it should really be parsed as an array of strings.

Reply:
It does match yes. `d2` is a substring of `r2d2` so `canFind` will discover it

Reply:
But this is not what we want, right? We want explicitly `+d2` I think.